i18n/src/main/res/values-zh-rCN/feat_setting.xml

```diff
@@ -72,58 +67,66 @@
     <string name="feat_setting_appearance_hint_edit_color">长按编辑颜色</string>
 
     <string name="feat_setting_tunneling">隧道模式</string>
-    <string name="feat_setting_tunneling_description">改善 4K/HDR 内容的播放</string>
+    <string name="feat_setting_tunneling_description">改善 4K/HDR 内容播放</string>
     <string name="feat_setting_label_backup">备份</string>
     <string name="feat_setting_label_restore">恢复</string>
 
-    <string name="feat_setting_canvas_dark">夜间</string>
+    <string name="feat_setting_canvas_dark">黑夜</string>
     <string name="feat_setting_canvas_light">日间</string>
     <string name="feat_setting_canvas_apply">应用</string>
     <string name="feat_setting_canvas_reset">重制</string>
 
     <string name="feat_setting_epg_clock_mode">12小时制节目单</string>
 
     <string name="feat_setting_remote_control">遥控器</string>
-    <string name="feat_setting_remote_control_description">开启遥控电视的能力</string>
+    <string name="feat_setting_remote_control_description">开启远端控制电视的功能</string>
 
     <string name="feat_setting_remote_control_tv_side">遥控器</string>
     <string name="feat_setting_remote_control_tv_side_description">允许智能手机控制你的电视</string>
 
     <string name="feat_setting_subscribe_for_tv">为电视订阅</string>
 
-    <string name="feat_setting_backing_up">备份所有播放列表和频道中</string>
-    <string name="feat_setting_restoring">恢复所有播放列表和频道中</string>
+    <string name="feat_setting_backing_up">正在备份所有订阅和频道</string>
+    <string name="feat_setting_restoring">正在恢复所有订阅和频道</string>
 
     <string name="feat_setting_follow_system_theme">跟随系统主题</string>
 
+    <string name="feat_setting_data_source_m3u" translatable="false">M3U</string>
+    <string name="feat_setting_data_source_epg" translatable="false">EPG</string>
+    <string name="feat_setting_data_source_xtream" translatable="false">Xtream</string>
+    <string name="feat_setting_data_source_emby" translatable="false">Emby</string>
+    <string name="feat_setting_data_source_dropbox" translatable="false">Dropbox</string>
+
     <string name="feat_setting_placeholder_basic_url">订阅地址</string>
     <string name="feat_setting_placeholder_username">用户名</string>
     <string name="feat_setting_placeholder_password">密码</string>
 
-    <string name="feat_setting_warning_xtream_takes_much_more_time">这可能会花费更久的时间</string>
+    <string name="feat_setting_warning_xtream_takes_much_more_time">可能需要花费更多时间</string>
 
     <string name="feat_setting_back_home">回到主页</string>
 
     <string name="feat_setting_always_replay">总是显示重播按钮</string>
-    <string name="feat_setting_player_panel">播放器控制台</string>
-    <string name="feat_setting_player_panel_description">竖屏状态下上滑播放器</string>
+    
+    <string name="feat_setting_player_panel">播放器面板</string>
+    <string name="feat_setting_player_panel_description">在竖屏状态时向上滑动以将其展开</string>
 
-    <string name="feat_setting_cache">边播边存</string>
-    <string name="feat_setting_cache_description">此选项可能导致无法正常播放</string>
+    <string name="feat_setting_cache">播放时缓存</string>
```

Comment:
restore

Reply:
```
<string name="feat_setting_cache">cache while playing</string>
    <string name="feat_setting_cache_description">this option may prevent normal playback</string>
    <string name="feat_setting_clear_cache">clear cache</string>
```
I just translate by these english wordings. Please advise or modify english version if needed.

Reply:
You can modify the English wordings.